.github/workflows/build.yml

```diff
@@ -213,6 +213,8 @@ jobs:
       with:
         submodules: false
         fetch-depth: 1
+    - name: Checkout frozen modules
+      run: git submodule update --init frozen
     - name: Get CP deps
       run: python tools/ci_fetch_deps.py docs ${{ github.sha }}
     - name: CircuitPython version
```

Comment:
If there's a reason not to roll this step into `ci_fetch_deps` can you let us know what it is? Superficially, it looks like `frozen` should just be added to the list in that file:
```patch
 elif target == "docs":
-    submodules = ["extmod/ulab/"]
+    submodules = ["extmod/ulab/", "frozen"]
 elif target == "mpy-cross-mac":
```
and if the data is needed during `create_website_pr.yml` it would need an added call to `ci_fetch_deps.py`. If it re-uses the same `docs` key that's fine, or you could add another key.

The reason I suggest this is so that someone who wants to build the docs locally can follow the documented method of using `ci_fetch_deps.py docs` rather than having to run an additional step. It also keeps the CI scripts tidier.